config/config.go

```diff
@@ -25,6 +25,58 @@ import (
 	tracing "github.com/uber/jaeger-client-go/config"
 )
 
+// Config number limitations
+const (
```

Comment:
We should document them in config.yaml.template and docs repos.

Reply:
Let's check these if these const is suitable or not, then I'll change the template and docs.